build/Dependencies.props

```diff
@@ -17,6 +17,7 @@
     <GoogleProtobufPackageVersion>3.5.1</GoogleProtobufPackageVersion>
     <LightGBMPackageVersion>2.2.1.1</LightGBMPackageVersion>
     <MicrosoftMLScoring>1.1.0</MicrosoftMLScoring>
```

Comment:
`MicrosoftMLScoring` should be removed now that it is no longer used.

Reply:
Removed.

---
In reply to: [236345353](https://github.com/dotnet/machinelearning/pull/1717#discussion_r236345353) [](ancestors = 236345353)